lib/mirage.ml

```diff
@@ -443,7 +443,9 @@ let archive_conf = impl @@ object
     method name = "archive"
     method module_name = "Tar_mirage.Make_KV_RO"
     method packages =
-      Key.pure [ package ~min:"0.7.0" ~ocamlfind:["tar.mirage"] "tar-format" ]
+      Key.pure [
+        package ~min:"0.7.0" ~ocamlfind:["tar-format.mirage"] "tar-format"
+      ]
```

Comment:
FWIW to me it looks like the `tar` package/ocamlfind got renamed yet another time... is that correct @djs55?  I suspect the current package needed for `tar` and MirageOS is `tar-mirage` (both ocamlfind and opam name), which itself depends on `tar`.